src/k8s/cmd/k8s/k8s_bootstrap.go

```diff
@@ -127,6 +128,14 @@ func newBootstrapCmd(env cmdutil.ExecutionEnvironment) *cobra.Command {
 				}
 			}
 
+			if bootstrapConfig.PodCIDR != nil && bootstrapConfig.ServiceCIDR != nil {
```

Comment:
Just out of curiosity, is it possible that only one of them is non-nil?

Reply:
If one of them is non-nil, then we do not need to check if there is an overlap (because the other is nil), so this is not an issue.